### ISSUE TITLE
Online Meetings Listing/Sorting

### ISSUE THREAD

**Issue by GCITechSC** [code4recovery/12-step-meeting-list#349]
Hi, I've filled out my Intergroups CSV, and right now in the middle of trying to change some of what is listed for online meetings.  I'm using the Elation Theme in Wordpress under GoDaddy.

What we're finding is if we search for an Online Meetings, Any Day, we're only getting the "Ladie's Night" which is a Google Meet.  Nothing else is coming up, as I've tried to change the "Brown Bag" on Monday to where I think the information should be, but it's still not showing in the search as an "Online Meeting". 

As always, your help is appreciated.

[GCI Meetings work.xlsx](https://github.com/code4recovery/12-step-meeting-list/files/6535453/GCI.Meetings.work.xlsx)

Scott

**Comment by tech2serve** [comment]
The Address for all meetings must be filled out with a geocodable address. You cannot use "Online Meeting" as an address. Likewise, "Zoom" shouldn't be used as a Location because that field should be unique to the address. 

Because a lot of users requested it, the address for an online meeting can be a city and state (thus making it approximate). 

If this doesn't resolve your problem, please let me know and I'll reopen the issue.

**Comment by GCITechSC** [comment]
Ok, this didn't work.  As my meeting list is now, even the ladies night doesn't show up with others on those specific days.  I do understand what you're saying about changing all of that.  I did change both one of the Ladies Nights, and The Tuesday for Brown Bag.  Brown Bag is still not showing up as "Closed, Online Meeting".  If you need me to send file again, please let me know.  Thanks.  

Scott

**Comment by joshreisner** [comment]
Hey Scott, I was able to import this CSV based on your spreadsheet pretty well. I made the following changes:

1. like @tech2serve suggested, set address to Columbia, SC for all online meetings
2. removed Online Meeting from types, because this gets set automatically when the plugin sees a conference URL it recognizes
3. lowercased Zoom.us in the URLs to zoom.us (👈 important)

There are several meetings that don't have a conference URL, those won't be considered "Online" meetings by the plugin, FYI

[GCI Meetings - GCI Meetings.csv](https://github.com/code4recovery/12-step-meeting-list/files/6543123/GCI.Meetings.-.GCI.Meetings.csv)

Hopefully this works for you.
